num_params:  402437
BARRIER DEADLOCK DURING INITIALIZATION CREATING RANDOM POINT TO ALLEVIATE!
Episode 1
Episode reward: 59.79999999999984
Episode steps: 81
Episode collisions: 0
Episode boundary: 0
Episode distance travelled: [5.93729613 6.53179638 5.60998316 5.5815388 ]

[Reward] Mean: 59.79999999999984, Standard Deviation: 0.0
[Steps] Mean: 81.0, Standard Deviation: 0.0
[Collisions] Mean: 0.0, Standard Deviation: 0.0
[Boundary] Mean: 0.0, Standard Deviation: 0.0
[Dist] Mean: [5.93729613 6.53179638 5.60998316 5.5815388 ], Standard Deviation: 0.382480861921534
Exception ignored in: <module 'threading' from '/usr/lib/python3.8/threading.py'>
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 1388, in _shutdown
    lock.acquire()
KeyboardInterrupt: 
